<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="windowTitle">
      <string>Clicker Game</string>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QLabel" name="lblEnemy">
            <property name="text">
              <string>Enemy: Goblin</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QLabel" name="lblHealth">
            <property name="text">
              <string>Health: 100</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="btnAttack">
            <property name="text">
              <string>Attack!</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QLabel" name="lblGold">
            <property name="text">
              <string>Gold: 0</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="btnUpgrade">
            <property name="text">
              <string>Upgrade (Cost: 100)</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
</ui>
